app/scripts/controllers/reviewRegistration.js

```diff
@@ -68,6 +68,105 @@ angular.module('confRegistrationWebApp')
       return _.find($scope.blocks, {id: blockId});
     };
 
+    var setRegistrationAsCompleted = function() {
+      registration = angular.copy(registration);
+      registration.completed = true;
+
+      RegistrationCache.update('registrations/' + registration.id, registration, function () {
+        RegistrationCache.emptyCache();
+        if(conference.registrationCompleteRedirect){
+          $window.location.href = conference.registrationCompleteRedirect;
+        }else{
+          $route.reload();
+        }
+      }, function (data) {
+        $scope.currentRegistration.completed = false;
+        $scope.submittingRegistration = false;
+        modalMessage.error(data.error ? data.error.message : 'An error occurred while submitting your registration.');
+      });
+    };
+
+    ////// START EVENT-433 //////
+
+    //Use $http.get using current registrant to get endPointResult and endPointSpouseInfo
+    var endPointResult = 200;
+    var endPointSpouseInfo = {firstName: 'Jane', lastName: 'Worner', email: 'janeworner@outlook.com'};
+
+    //If get 200 back from endpoint then show Include Spouse button
+    //else if 204 do not show button
+    if(endPointResult === 200) {
+      $scope.includeSpouseButtonVisible = true;
+    }
+    if(endPointResult === 204 || localStorage.getItem('includeSpouseButtonVisible') === 'false') {
+      $scope.includeSpouseButtonVisible = false;
+    }
+
+    //Click event for 'Include Spouse' button
+    $scope.includeSpouseInRegistration = function () {
+
+      //Parameters needed to retrieve all current registrants for conference.id
+      var queryParameters = {
+        block: [],
+        //page: 1,
+        //limit: 20,
+        orderBy: 'last_name',
+        order: 'ASC',
+        filter: '',
+        filterPayment: '',
+        filterRegType: '',
+        includeCheckedin: 'yes',
+        includeWithdrawn: 'yes',
+        includeIncomplete: 'yes'
+      };
+
+      //If false then spouse found has not been regitered yet
+      var spouseAlreadyRegistered = false;
+
+      //If spouse does not exists in registry then enter a new registrant based on return from endPointSpouseInfo
+      //else no need to register because already registered
+      var updateRegistrants = function() {
+        if (!spouseAlreadyRegistered) {
+          var registrationId = uuid();
+          $scope.currentRegistration.registrants.push({
```

Comment:
OK, fair enough. 😄 The only way this this would matter is if the ERT ever supported registering multiple people (like maybe a parent and children?) in the same registration, which it currently does not as far as I can tell.